tests/shell/ShellCheckBearTest.py

```diff
@@ -17,7 +17,27 @@
 echo $X
 """
 
+valid_with_ignore_rules_file = """
+#!/usr/bin/env sh
+echo "Path is $PATH"
+cd /home
+for dir in */
+do
+  cd "$dir"
+  convert index.png index.jpg
+  cd ..
+done
+"""
```

Comment:
Let's split this into two smaller test cases. The first one would test `SC2164`, the second one will test `SC2103`. The first test file can then be as simple as:

```
#!/usr/bin/env sh
cd
```
Please figure out the minimal file for the second error then.

Also, you could add cross-test case: exclude one error but feed the other file and check that the other error is still triggered.